guides/searcher-onboarding.md

```diff
@@ -124,48 +198,27 @@ MEV-Geth, MEV-Relay and all the code searchers interact with is open-source and
 
 We've also released a publicly accessible API blocks.flashbots.net for displaying Flashbots blocks and txs, and will be releasing live data visualizations useful to searchers in the coming weeks.
 
-
 ### Can I use Flashbots concurrently with any other similar systems mentioned above? 
 Yes! As a searcher you want to maximize the hashrate you're exposed to and there is no reason you can't submit your trades to Flashbots and another system in parallel.
 
 One could also imagine a dual system that submits txs to both the traditional Ethereum mempool and Flashbots, with bot logic conditional on one or the other landing.
 
 ### Why should I adopt Flashbots?
-Flashbots is immediately useful to any searcher who has ever lost PGAs or lost money on unsuccessful trades that still cost gas.
+Flashbots is immediately useful to any user who has ever lost priority gas auctions ("PGAs") or lost money due to unsuccessful trades that still cost gas.
```

Comment:
... or lost an opportunity to a generalized front-runner (although it is a subset of PGA, it might be interesting to some users). Nothing is worse than putting in a few hours of work on a brand new opportunity to lose it to a bot that doesn't even understand what it's doing.